data/csv/timestamp_timezone.csv

```diff
@@ -0,0 +1,6 @@
+time,description
+2020-01-01T00:00:00,midnight local
+2020-01-01T00:00:00-08,midnight in San Francisco
+2020-01-01T00:00:00+00,midnight UTC +00
+2020-01-01T00:00:00Z,midnight UTC Z
+2020-01-01T00:00:00+08:00,midnight in Taipei
```

Comment:
Always fun to include fractional hours for this stuff (Asia/Kolkata +05:30:00, Pacific/Chatham +13:45, Canada/Newfoundland -03:30:00)

Reply:
Just to check if I got your comment right,

I should add:
```
2020-01-01T00:00:00+05:30:00,Asia/Kolkata 
2020-01-01T00:00:00+13:45, Pacific/Chatham
2020-01-01T00:00:00 -03:30:00,Canada/Newfoundland
```

to the CSV file, right?